bokun.yaml

```diff
@@ -6525,44 +6945,66 @@ definitions:
         type: string
   StartTimeDto:
     properties:
-      duration:
+      id:
+        type: integer
+        format: int64
+        description: 'Specifies ID of the starting time.'
+      label:
+        type: string
+        description: 'Not working. Specifies label for this starting time.'
+      hour:
         type: integer
         format: int32
-      durationDays:
+        description: 'Specifies hours of this starting time.'
+      minute:
         type: integer
         format: int32
-      durationHours:
+        description: 'Specifies minutes of this starting time.'
+      overrideTimeWhenPickup:
+        type: boolean
+        description: 'Specifies if this starting time is overriden by pick up time.'
+      pickupHour:
         type: integer
         format: int32
-      durationMinutes:
+        description: 'Specifies start time hours when different time for pick-up is selected.'
+      pickupMinute:
         type: integer
         format: int32
+        description: 'Specifies start time minutes when different time for pick-up is selected.'
       durationType:
         type: string
-      durationWeeks:
+        description: 'Deprecated. Should be used durationMinutes, durationHours, durationDays, durationWeeks.'
+      voucherPickupMsg:
+        type: string
+        description: 'Not working. Pick-up message on the ticket'
```

Comment:
Remove comment about not working

Reply:
Should I also fill bug report or you found it working?

Reply:
File a bug.